securityhub-forwarder/sam/requirements.txt

```diff
@@ -1 +1 @@
-boto3==1.9.66
+boto3==1.27.1
```

Comment:
Can this be upgraded to latest version - 1.28.74

Reply:
I followed AWS's [recommendations](https://docs.aws.amazon.com/lambda/latest/dg/lambda-python.html)